src/test/java/com/wikia/webdriver/common/core/api/ApiCall.java

```diff
@@ -43,14 +44,23 @@ protected ApiCall() {}
    */
   abstract protected ArrayList<BasicNameValuePair> getParams();
 
+  /**
+   * This enables passing username as string instead from ENUM.
+   *
+   * @return Username to log in.
+   */
+  abstract protected String getUserName();
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Reorder the modifiers to comply with the Java Language Specification. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://sonar-s4/coding_rules#rule_key=squid%3AModifiersOrderCheck)